### ISSUE TITLE
Is it possible to add customer attributes to the table?

### ISSUE THREAD

**Issue by gibbons1984** [alaarihan/woo-variations-table#38]
I have a custom attribute set up for 'Power' in the product variations, is it possible to display this in the table? Thanks

**Comment by alaarihan** [comment]
Yes, its' possible, just make sure you select the option from the product edit page for the attribute " Visible on the product page" beside enabling the attributes column from the plugin settings page.
